layouts/partials/fragments/contact.html

```diff
@@ -107,34 +107,34 @@ <h2>{{- . | markdownify -}}</h2>
               </div>
               <div class="col-lg-12 text-center">
                 <div class="generic-success text-success d-none" id="generic-success">
-                  {{ with $data.message.success }}
+                  {{ with $contact.message.success }}
                     {{- . | markdownify -}}
                   {{ else }}
                     {{- i18n "contact.defaultGenericSuccess" -}}
                   {{ end }}
                 </div>
                 <div class="captcha-error text-danger d-none" id="captcha-error">
-                    {{- i18n "contact.defaultCaptchaError" }} {{ $data.email -}}.
+                    {{- i18n "contact.defaultCaptchaError" }} {{ $contact.email -}}.
                 </div>
                 <div class="generic-error text-danger d-none" id="generic-error">
-                  {{ with $data.message.error }}
+                  {{ with $contact.message.error }}
                     {{- . | markdownify -}}
                   {{ else }}
-                    {{- i18n "contact.defaultGenericError" }} {{ $data.email -}}.
+                    {{- i18n "contact.defaultGenericError" }} {{ $contact.email -}}.
                   {{ end }}
                 </div><br />
                 <noscript>
                   <div class="text-danger py-2" id="js-error">
-                    {{- i18n "contact.defaultJsError" }} {{ $data.email -}}.
+                    {{- i18n "contact.defaultJsError" }} {{ $contact.email -}}.
                   </div>
                 </noscript>
                 {{ with $.Site.Params.contact.captcha }}
-                    <div class="g-recaptcha" data-sitekey="{{ .sitekey }}" data-size="invisible" data-badge="inline" data-callback="onContactCaptcha"></div>
+                    <div class="g-recaptcha" contact-sitekey="{{ .sitekey }}" contact-size="invisible" contact-badge="inline" contact-callback="onContactCaptcha"></div>
```

Comment:
This change breaks the contact form. Search and replace did replace a bit too much here ;)